src/shared/selectors/accounts.js

```diff
@@ -99,7 +99,7 @@ export const selectLatestAddressFromAccountFactory = (withChecksum = true) =>
  **/
 export const getSelectedAccountMeta = createSelector(
     selectAccountInfo,
-    (account) => account.meta || { type: 'keychain' },
+    (account) => get(account, 'meta'),
```

Comment:
If you want to preserve the existing functionality where it will return `'keychain'` as a fallback, :

```suggestion
    (account) => get(account, 'meta', { type: 'keychain' }),
```

Reply:
Returning `undefined` would be a more correct behaviour.